packages/lit-node-client-nodejs/src/lib/helpers/get-signatures.ts

```diff
@@ -145,22 +150,21 @@ export const getSignatures = async <T>(params: {
   ];
 
   if (allKeys.length !== initialKeys.length) {
-    throwError({
-      message: 'total number of valid signatures does not match requested',
-      errorKind: LIT_ERROR.NO_VALID_SHARES.kind,
-      errorCode: LIT_ERROR.NO_VALID_SHARES.code,
-    });
+    throw new NoValidShares(
+      {},
+      'total number of valid signatures does not match requested'
+    );
   }
 
   // -- combine
-  for (var i = 0; i < allKeys.length; i++) {
+  for (const key of allKeys) {
```

Comment:
Is there a benefit of using an iterator over just accessing the elements of the array?

Reply:
We have an [eslint rule](https://typescript-eslint.io/rules/prefer-for-of/) to prefer that. I don't think there is a considerable difference between the two

This change is just to lower the IDE warnings so someday, that can help us